source/releasenotes/2024-06-13-quicksilver-examples-new-relic.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Necessary updates to New Relic scripts in quicksilver-examples repository."
+published_date: "2024-06-13"
+categories: [tools-apis, action-required]
+---
+
+The internals of the integration between Pantheon and New Relic are changing due to some updates we should do on Pantheon's side to stay up-to-date with updates on New Relic side. These changes will be transparent for most of our users except:
+
+- Those who got an email warning you about changes in your New Relic account
+- Those who are using [quicksilver](https://docs.pantheon.io/guides/quicksilver) to interact with New Relic API.
```

Comment:
```suggestion
The internals of the integration between Pantheon and New Relic are changing due to some updates we should do on Pantheon's side to stay up-to-date with updates on New Relic side. These changes will be transparent for most of our users except:

- Those who are using [quicksilver](https://docs.pantheon.io/guides/quicksilver) to interact with New Relic API.
```
Removing this line because we didn't end up emailing customers about changes to New Relic.